### ISSUE TITLE
[Question] Why is img_scaling_factor set to 1.0?

### ISSUE THREAD

**Issue by JieZou1** [yhenon/keras-frcnn#203]
That means the pixel values are normalized to approximately [-128, 127]. Shouldn't we normalize it into approximately [-1, 1], for easy converge? Many thanks.

**Comment by concerttttt** [comment]
Scaling is an option but not a must, normalize it would be helpful to the model convergence, in some sense.

The point here is to remove redundant computation caused by mean value, there is no much difference for computers to calculating at the magnitude of 128 or 1, both would be super fast.

**Comment by JieZou1** [comment]
Okay, thanks for your info. However, just FYI, in my own data set, when I do not normalize the pixel value to [-1,1], the training does not converge. After normalization, the training happily converges. So, even though theoretically, they should be the same, normalization to [-1,1] may make better sense from numerical computing point of view. Could anyone explain it in more details? Many thanks.

**Comment by concerttttt** [comment]
Yes, you should normalize your data before training, if this is the answer you expected.

Many tricks and parameters will affect the data training process, data normalization is one option in pre-processing stage, others include the distribution of your data, data standardization, whitening, PCA sometimes, initialization, batch normalization... data normalization was proved to be useful in many cases,  but it's hard to claim normalization is the only reason and only solution to train up a model, which means you can take it as a sufficient condition but not a necessary condition. 

Do it if you do believe it's effective, it won't bite anyway.

You might want to look at chapter 7 & 8 in [Deep Learning Book](http://www.deeplearningbook.org/), and this [paper by Yann Lecun](http://yann.lecun.com/exdb/publis/pdf/lecun-98b.pdf)

**Comment by JieZou1** [comment]
Okay, great! Thank you very much for your information.
